app/models/work.rb

```diff
@@ -470,8 +470,7 @@ def format_date_for_solr(date)
   # S3QueryService object associated with this Work
   # @return [S3QueryService]
   def s3_query_service
-    mode = approved? ? "postcuration" : "precuration"
-    @s3_query_service ||= S3QueryService.new(self, mode)
+    @s3_query_service ||= S3QueryService.new(self, files_bucket_name)
```

Comment:
Moved the logic for the location to its own method `files_bucket_name` (see below)